modules/avformat/avformat.c

```diff
@@ -173,6 +179,21 @@ static int open_codec(struct stream *s, const struct AVStream *strm, int i,
 		return ENOMEM;
 	}
 
+    if (avformat_hwdevice != AV_HWDEVICE_TYPE_NONE)
+    {
```

Comment:
please move curly bracket to end of previous line.

also please check the indentation, should be TAB(s)